templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -231,9 +231,9 @@
 							{{.locale.Tr "repo.issues.new.closed_projects"}}
 						</div>
 						{{range .ClosedProjects}}
-							<a class="item muted sidebar-item-link" data-id="{{.ID}}" data-href="{{$.RepoLink}}/projects/{{.ID}}">
+							<a class="item muted sidebar-item-link" data-id="{{.ID}}" data-href="{{.Link}}">
 								{{svg "octicon-project" 18 "mr-3"}}
-								{{.Title}}
+								{{if .IsOrganizationProject}}org: {{end}}{{.Title}}
```

Comment:
Translate `org: ` ?
or should we use icon .... like we do with teams for requested reviewers?

Reply:
Different icon or different color of the same icon?

Reply:
Use a different icon for repo project and org project.